### PR TITLE
added shit please dm me on discord @fifthwit for bounty ty

### PR DESCRIPTION
**Pull request by FifthWit**
did this for the bounty adds all the docs I thought were relevant and the kajigs, any that werent dead ig, probably a lot of them were but whatever discord is @fifthwit

### CODE REVIEW

**Comment by ProgrammerIn-wonderland** [comment]
@TBNRAubrey review?

**Review by r58Playz — CHANGES_REQUESTED** [review]
Simply a copy paste of the readmes. Even the curltransport <-> twisp swap is still here. Don't merge

**Comment by FifthWit** [comment]
> Simply a copy paste of the readmes. Even the curltransport <-> twisp swap is still here. Don't merge

thats the point genius, to centralize shit like kajigs lmao

**Comment by wearrrrr** [comment]
Yeah, please keep in mind that MDX has more features than MD, and those are generally expected to be used here.

**Comment by r58Playz** [comment]
> > Simply a copy paste of the readmes. Even the curltransport <-> twisp swap is still here. Don't merge
> 
> thats the point genius, to centralize shit like kajigs lmao

image
This doesn't document anything.

File: docs/kajigs/KIOSK_Exploit_DE-LICENSED_Original_.mdx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+- Any kiosk app installed
+- [for v77<=x<=v85] OOBE/fresh install
+- Sign out
+- Turn Wi-Fi off
+- Enable ChromeVox (ctrl+alt+z)
+- Open a kiosk app
+- Spam Search+o+k (you do have to re-click o and k)
+- While spamming click "Diagnose"
+
+Note: The exploit should work consistently v76 or below but it's still possible to pull off on v85 or below, but only once (right after you powerwash).
```
Comment (crazycrystals) [inline docs/kajigs/KIOSK_Exploit_DE-LICENSED_Original_.mdx:26]: did you even try to use the capabilities of MDX?
```
:::note
TEXT
:::
```

File: docs/kajigs/Omada_DNS.mdx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+## Notes
+- GG is working once again! this will likely be patched again soon.
+- Lightspeed is semi-patched. It takes a cache of blocked websites. Simply keep the DNS on for 10 days or more and the cache will be cleared.
+- Securly schools have settings disabled because of caudns 💀
+- iboss is patched
+- Contentkeeper is patched, both the network level version and ck-express. 
+- If you're already using omadadns, continue to do so, and DO NOT connect to a wifi network without it, unless your extensions may update and be patched.
+- This does NOT work on network level filters such as fortinet, iboss, or some cisco configurations
+- The full ext removal method breaks locked mode quizes, to fix them revert the DNS then remove your user and add it back.
+- This very commonly breaks locked mode quizes. To fix this, turn off omadadns and then restart your chromebook, go to chrome://policy, and then refresh policies. Wait for it to refresh, wait five minutes, then do another restart. If this doesn't fix, turn off omadadns and powerwash your chromebook or remove and readd your user.
```
Comment (crazycrystals) [inline docs/kajigs/Omada_DNS.mdx:31]: :::noteIssues
all that
:::
Also, actually link to other kajigs when you reference them. A wiki should allow you to go through it without having to actively search to find something it references.

File: docs/kajigs/Root_Escalation.mdx
Comment (crazycrystals) [inline docs/kajigs/Root_Escalation.mdx:1]: this shouldnt be on the wiki. Its using the forum function. Also, you left the discord message link there

File: docs/kajigs/Shimboot.mdx
Comment (crazycrystals) [inline docs/kajigs/Shimboot.mdx:1]: vk6 will KILL YOU for not giving him credit

File: docs/kajigs/Skiovox_Breakout.mdx
Comment (crazycrystals) [inline docs/kajigs/Skiovox_Breakout.mdx:1]: "\@akabutnice on discord"

oh shit and I realized your not doing double newlines so markdown is just not going to do newlines lmaoo

File: docs/kajigs/Skiovox_v125.mdx
Comment (crazycrystals) [inline docs/kajigs/Skiovox_v125.mdx:1]: tips as
:::noteTips
tips here
:::

File: docs/kajigs/Stealth_Lightspeed.mdx
Comment (crazycrystals) [inline docs/kajigs/Stealth_Lightspeed.mdx:1]: :::note
noteshere
:::

File: docs/kajigs/UserPolicy_Bypass_.mdx
Comment (crazycrystals) [inline docs/kajigs/UserPolicy_Bypass_.mdx:1]: remove your DAMN discord links
anyways im done reviewing you get the point

**Review by crazycrystals — COMMENTED** [review]
its just copy pasted bruh you didnt convert to a "wiki" format and you copied kajigs and repos that did not need to be copied in the first place (rootesc kajig)

**Comment by QuiteAFancyEmerald** [comment]
I'm sorry going to have to deny this one; it doesn't even include all the TN repos so your uh pasting was not effective.

**Pull request closed without merging**